lib/core/presentation/pages/event/event_detail_page/event_detail_base_page.dart

```diff
@@ -68,12 +69,21 @@ class _EventDetailBasePageView extends StatelessWidget {
                 orElse: () => '',
                 authenticated: (session) => session.userId,
               );
-          final isCohost = EventUtils.isCohost(event: event, userId: userId);
+          final eventUserRole =
+              context.watch<GetEventUserRoleBloc>().state.maybeWhen(
```

Comment:
What make sure this state will be available before event detail is fetched ? There will be "UI jump" if GetEventUserRole not done fetching yet